ping.go

```diff
@@ -497,6 +513,34 @@ type IcmpData struct {
 	Tracker int64
 }
 
+func (p *Pinger) checkWaitPacket() {
+	outPkt := &Packet{
+		IPAddr:  p.ipaddr,
+		Addr:    p.addr,
+		Seq:     p.sequence,
+		ReqTime: time.Now(),
+		Timeout: false,
+	}
+
+	p.waitPackets = append(p.waitPackets, outPkt)
```

Comment:
Won't this list will grow forever? Can you please implement something to keep it short?